docs/kubernetes/backup.md

```diff
@@ -29,6 +29,7 @@ The [CSI Snapshot](https://github.com/container-storage-interface/spec/blob/mast
     name: csi-filestore
     provisioner: filestore.csi.storage.gke.io
     parameters:
+      tier: enterprise
```

Comment:
Can you also fix any mention of supported tiers and add enterprise e.g https://screenshot.googleplex.com/Q6GvZxP3cosD5be